Userland/Libraries/LibWeb/Layout/GridFormattingContext.cpp

```diff
@@ -1169,7 +1169,7 @@ void GridFormattingContext::expand_flexible_tracks(AvailableSpace const& availab
     auto& tracks = dimension == GridDimension::Column ? m_grid_columns : m_grid_rows;
     auto& available_size = dimension == GridDimension::Column ? available_space.width : available_space.height;
 
-    auto find_the_size_of_an_fr = [&](Vector<GridTrack&> tracks, CSSPixels space_to_fill) -> CSSPixels {
+    auto find_the_size_of_an_fr = [&](Vector<GridTrack&> const& /*FIXME: Span*/ tracks, CSSPixels space_to_fill) -> CSSPixelFraction {
```

Comment:
could you add more elaborate description for this `FIXME` please?

Reply:
Ah yeah, wanted to make it a span, but that does not like references.
Should I make it a bigger fixme or remove it and keep it in the back of my head until span can do references?